jsonrpc/testsuite/transaction-eip1559.json

```
@@ -14,5 +14,6 @@
     "from": "0x0300000000000000000000000000000000000000",
     "blockHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
     "blockNumber": "0x1",
-    "transactionIndex": "0x2"
+    "transactionIndex": "0x2",
+    "type": "0x0"
```

Nitpick: 
since this is a dynamic fee tx, probably it would be more correct to put dynamic fee tx type

```suggestion
    "type": "0x02"
```

---

Doing this the marshaling is different. , any idea why this would happen: 
```        	            	- (string) (len=4) "type": (string) (len=4) "0x02",
        	            	+ (string) (len=4) "type": (string) (len=3) "0x2",
        	            	```